python/src/main/db.py

```diff
@@ -46,9 +46,27 @@ def get_items(self, email):
             items += row
         return items
 
+    def get_specific_item(self, email, name):
+        items = []
+        rows = self.cursor.execute("SELECT * FROM ITEM WHERE person_email = '%s' AND name = '%s'"
```

Comment:
Can we use an sql statement that only returns one value?

Reply:
The plan was for this to return just a single row

Reply:
have a google see if it can be done :)